### ISSUE TITLE
1.11 beta breaks ember-validations

### ISSUE THREAD

**Issue by bcardarella** [emberjs/ember.js#10689]
@wycats said these issues should be reported http://discuss.emberjs.com/t/is-the-six-week-release-cycle-too-frequent/7526/56

Ember Validations is not using any private API yet the bump from Ember 1.10 to 1.11 beta is causing failures: https://github.com/dockyard/ember-validations/issues/268

**Comment by stefanpenner** [comment]
lets wait to see if this is actually an ember bug. If it is, lets re-open. So-far it does not appear to be.

**Comment by rondale-sc** [comment]
This is a problem for us as well.  We ran into this in the 1.11 beta, but reverted assuming this would have been fixed by main release.  Now with the security issue this seems like a pretty big blocker.  I have been looking at this issue and associated issues.  It seems as if ember-data is _not_ to blame.  Though I could be wrong.

The error I'm receiving:

![](http://i.imgur.com/0CEdZfK.png)

And

---
![](http://i.imgur.com/TQYwsU0.png)


Versions:

Ember: 1.11.1
Ember Data: 1.0.0-beta.16.1

I think I'll post this to the emberjs ticket as well.
